components/typography/Label.tsx

```diff
@@ -13,11 +13,12 @@ const mapDesign = {
   S: 'text-sm not-italic font-semibold leading-none',
 };
 
-export const Label = ({ text, size }: ILabelProps) => {
+export const Label = ({ text, size, ...labelProps }: ILabelProps) => {
```

Comment:
I was thinking that the Label component is not part of the typography. I would move it to the components. 

The labels on the Typography section of figma refer to font size/line-height/font-weight definitions to be used on other html tags as h5/h6/p/span/ etc. AS subtitles. 
I would not use Label html tags somewhere else than in conjunction with an input html tag. 

If I would need to use some  Figma label within a text, I would use  for example a h5/h6  followed by a Paragraph with the text. 

What do you think about this?